plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -94,6 +94,7 @@ def launch(
             sweep_configs=sweep_configs,  # type: ignore
             task_function=launcher.task_function,
             singleton_state=Singleton.get_state(),
+            ray_init_cfg=launcher.ray_cfg.init,
```

Comment:
all the composed configurations is included in `sweep_configs`, I'd prefer not to do extra pickling for one of the configs.